docs/content/preview/explore/_index.md

```diff
@@ -12,151 +12,183 @@ aliases:
   - /preview/explore/planet-scale/
   - /preview/explore/cloud-native/orchestration-readiness/
 type: indexpage
+showRightNav: true
 ---
 
-<div class="row">
-  <div class="col-12 col-md-6 col-lg-12 col-xl-6">
-    <a class="section-link icon-offset" href="ysql-language-features/">
-      <div class="head">
-        <img class="icon" src="/images/section_icons/explore/fault_tolerance.png" aria-hidden="true" />
-        <div class="title">SQL features</div>
-      </div>
-      <div class="body">
-        SQL features in YSQL.
-      </div>
-    </a>
-  </div>
+The Explore section walks you through YugabyteDB's core features, with examples.
 
-  <div class="col-12 col-md-6 col-lg-12 col-xl-6">
-    <a class="section-link icon-offset" href="fault-tolerance/macos/">
-      <div class="head">
-        <img class="icon" src="/images/section_icons/explore/fault_tolerance.png" aria-hidden="true" />
-        <div class="title">Fault tolerance</div>
-      </div>
-      <div class="body">
-        Resilient and self-healing.
-      </div>
-    </a>
-  </div>
+Most examples demonstrating database features such as API compatibility can be run on a single-node cluster on your laptop or using the free Sandbox cluster in YugabyteDB Managed. More advanced scenarios use a multi-node deployment. Refer to [Set up your environment](#set-up-your-environment) for instructions on creating clusters to run the examples.
 
-  <div class="col-12 col-md-6 col-lg-12 col-xl-6">
-    <a class="section-link icon-offset" href="linear-scalability/">
-      <div class="head">
-        <img class="icon" src="/images/section_icons/explore/linear_scalability.png" aria-hidden="true" />
-        <div class="title">Horizontal scalability</div>
-      </div>
-      <div class="body">
-        Reliable, rapid scaling for both writes and reads.
-      </div>
-    </a>
-  </div>
+The following table describes the YugabyteDB features you can explore, along with the setup (single node or multi node cluster) required to run the examples.
```

Comment:
Suggestion:
The following table describes the YugabyteDB features you can explore, along with the setup required to run the examples (single-node or multi-node cluster).